portfolio/src/main/java/com/google/sps/servlets/ListImagesServlet.java

```diff
@@ -24,6 +25,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         Storage storage = StorageOptions.newBuilder().setProjectId(projectID).build().getService();
         Bucket bucket = storage.get(bucketName);
         Page<Blob> blobs = bucket.list(); //Is this an actual list?
```

Comment:
You could look at the Javadoc for `Page<T>` and see if it implements `java.util.List`

Reply:
Yeah, I forgot to remove that comment, but from what I got it was kind of a list? It says "A page object wraps an API list method response" so a wrapper for a list? You can iterate it using `Iterable` though.